File: problemas/iniciante/1131/1131.swift
import Foundation

var inter = 0
var gremio = 0
var empate = 0
var codigo = 0
while (codigo != 2) {
  let numeros = readLine(strippingNewline: true)!.components(separatedBy: " ").compactMap { Int 
  let i = numeros[0]
  let g = numeros[1]

  if (i > g) {
    inter += 1
  } else if (i == g) {
    empate += 1
  } else {
    gremio += 1
  }

  codigo = 0
  while (codigo != 1 && codigo != 2) {
    print("Novo grenal (1-sim 2-nao)")
    codigo = Int(readLine(strippingNewline: true)!)!
  }
}

print(String(format: "%d grenais", inter + gremio + empate))
print(String(format: "Inter:%d", inter))
print(String(format: "Gremio:%d", gremio))
print(String(format: "Empates:%d", empate))
if (inter > gremio && inter > empate) {
  print("Inter venceu mais")
} else if (gremio > inter && gremio > empate) {
  print("Gremio venceu mais")
} else {
  print("Nao houve vencedor")

File: problemas/iniciante/1131/1131.kt
import java.util.*

fun main(args: Array<String>) {
  val sc = Scanner(System.`in`)

  var inter = 0
  var gremio = 0
  var empate = 0
  while (true) {
    val i: Int = sc.nextInt()
    val g: Int = sc.nextInt()

    if (i < g) {
      gremio += 1
    } else if (i == g){
      empate += 1
    } else {
      inter += 1
    }

    var codigo = 0
    while (codigo != 1 && codigo != 2){
      println("Novo grenal (1-sim 2-nao)")
      codigo = sc.nextInt()
    }

    if(codigo == 2){
      break
    }
  }

  println(String.format("%d grenais", inter + gremio + empate))
  println(String.format("Inter:%d", inter))
  println(String.format("Gremio:%d", gremio))
  println(String.format("Empates:%d", empate))

  if(inter > gremio && inter > empate){
    println("Inter venceu mais")
  }else if(gremio > inter && gremio > empate){
    println("Gremio venceu mais")
  } else {
    println("Nao houve vencedor")
  }
}
